src/werkzeug/debug/shared/debugger.js

```diff
@@ -1,3 +1,25 @@
+function fadeOut(el){
+    el.style.opacity = 1;
+
+    (function fade() {
+        if ((el.style.opacity -= .1) < 0) el.style.display = "none";
+        else requestAnimationFrame(fade);
+    })();
+};
+
+function fadeIn(el, display){
+    el.style.opacity = 0;
+    el.style.display = display || "block";
+
+    (function fade() {
```

Comment:
Just curious, why is this in an IIFE? Can we remove the name `fade`?

Reply:
it calls itself in the `requestAnimationFrame` function below